templates/careers/apply.html

```diff
@@ -45,7 +45,7 @@ <h3>Roles applying for:</h3>
             <label for="CheckMe3">Software Engineer - MAAS</label>
           </div>
           <div class="col-3 u-align--right">
-            <button class="p-button has-icon">
+            <button class="p-button has-icon" onclick="document.querySelector('.p-modal').classList.remove('u-hide');">
```

Comment:
This would be better to call a function so you can null check the modal in case someone removes/edits it in the future it will error.